packages/core/modules/index.d.ts

```diff
@@ -1369,6 +1369,7 @@ export interface BehaviourSettings {
   maxNumberOfCases?: Number;
   showErrorMessage?: boolean;
   convertableWidgets?: TypedMap<Array<string>>;
+  jsonLogicExportPreserveGroups?: boolean;
```

Comment:
Also please add description of new setting in https://github.com/ukrbublik/react-awesome-query-builder/blob/master/CONFIG.adoc#configsettings